src/client/feed/Feed.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const Feed = ({ feed }) => {
+  return (

```

Comment:
If not doing anything in the function apart from returning react component, can we get rid of the return keyword and { braces... just slightly cleaner